java/code/src/com/suse/manager/webui/controllers/DownloadController.java

```diff
@@ -86,6 +87,7 @@
         private String release;
         private String epoch;
         private String arch;
+        private String checksum;
```

Comment:
I'd suggest using `Optional<String>` here, instead of using `""` as the implicit absence value.